app/load_data/load_ndcs.py

```diff
@@ -1,11 +1,16 @@
 import requests
 import sys
 import json
-from connect_db import Drugs, Routes, ProductTypes, PharmClasses
-from helpers import startLogging
+from load_data.models import Drugs, Routes, ProductTypes, PharmClasses
+from helpers.helpers import startLogging
+from pathlib import Path
 
 logger = startLogging('load_ndcs')
 
+def find_json():
+    p = Path('.')
+
+
```

Comment:
Cant tell if this is being used anywhere.  Is this something for future implementation? Assuming auto discovery of data file

Reply:
That is complete garbage I missed during clean up.  Nice catch!